chart/templates/_commonChecks.tpl

```diff
@@ -161,6 +216,57 @@ Return customer values to use in preflights and support-bundle
     value: {{ include "carto.postgresql.databaseName" . }}
   - name: WORKSPACE_POSTGRES_USER
     value: {{ include "carto.postgresql.user" . }}
+  - name: WORKSPACE_TENANT_ID
+    value: {{ .Values.cartoConfigValues.selfHostedTenantId | quote }}
```

Comment:
jummm, Today I discovered that quotes could broke things. We need to add validators to the buckets and some other places to avoid blank spaces (or properly trim all inputs)

Reply:
I think that in this case it's not dangerous 🤔 At backend level we're taking it into account so I think that there is no problem